### DISCUSSION TITLE
Future of xESMF

### DISCUSSION THREAD

**Discussion by marstonsward** [pangeo-data/xESMF#320] [General]
Hi,

I would like to suggest xESMF for a regridding tool in a production code, primarily for it conservative regridding but I'm seeing a lot of old issues that are not closed, i.e., resolved. Is this project dying or dead?

**Comment by raphaeldussin** [comment]
we're all dying but at a very slow pace hopefully...
xESMF is a fairly mature project so there is not a lot of commits but new functionalities are added on a regular basis.
This code is maintained by a community of very busy **volunteers** so we work on the project as time allows.

**Reply by marstonsward** [reply]
Understood. But there should still be some standards maintained in order for the code not to suffocate, so to speak. Issues that have been opened over a year do raise some concerns for new users. 
I've used this regridder when it was part of NCL, and I was hoping to see it back in business but maybe wishful thinking. I'm afraid I have to find something else to use, sadly.

**Reply by Thomas-Moore-Creative** [reply]
I think a great standard to start with is for commercial & Government organisations to budget support for the open source software development that powers their projects and client delivery?  Mia Culpa we are not good at it - but I'm not in charge ; - )

And, thanks for the xESMF dev work from the volunteers.
